.github/workflows/remind-docs-and-tests.yml

```
@@ -9,8 +9,9 @@ jobs:
       - uses: wow-actions/auto-comment@v1
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            permissions: read-all|write-all
```

@elongl you mean something like 

```suggestion
      permissions: read-all|write-all
      - uses: wow-actions/auto-comment@v1
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```